configs/node.js

```diff
@@ -0,0 +1,45 @@
+/** @type import("eslint").Linter.Config */
+module.exports = {
+  env: {
+    es2021: true,
+    node: true,
+  },
+  parser: "@typescript-eslint/parser",
+  parserOptions: {
+    ecmaVersion: "latest",
+    sourceType: "module",
+  },
+  extends: [
+    "standard",
+    "plugin:@typescript-eslint/recommended",
+    "plugin:prettier/recommended",
+  ],
+  plugins: ["@typescript-eslint", "simple-import-sort"],
+  rules: {
+    "prettier/prettier": [
+      "error",
+      {
+        printWidth: 100,
+        tabWidth: 2,
+        singleQuote: true,
+        trailingComma: "all",
+        arrowParens: "always",
+        semi: true,
+      },
+    ],
```

Comment:
question: would it be possible to keep a base.js file with the parts that are common to both configs, like this?

Reply:
For sure, we can create a base file for the configs. Do you think we need to export the base config as well? I don't think we'll use the base config in any project, unless we can't use the react/node variants for some reason.

Reply:
It would be a nice to have, but I wasn't worried about exporting it, I was mostly thinking about sharing code (and removing duplicates)

Reply:
Cool, I created a `base.js` file with base configuration for the other ones.